examples/gateway/anthropic/README.md

```diff
@@ -0,0 +1,27 @@
+## Example route configuration for Anthropic
+
+To set up your MLflow configuration file, include a single route for the completions endpoint as follows:
```

Comment:
Can we just say `see examples/gateway/anthropic.yaml`?

Reply:
yep! changed all 3 to just redirect to the config files